repo/packages/C/confluent-replicator/3/config.json

```diff
@@ -0,0 +1,62 @@
+{

```

Comment:
@dbtucker directory version here doesnt affect the version of the package in Universe. It is for us to keep track of how many version updates were made starting with directory version 0. Please change it for `confluent-replicator`